dephell/converters/sdist.py

```diff
@@ -204,7 +194,9 @@ def _write_content(self, tar, path: str, content) -> None:
 
     # poetry/masonry/builders/sdist.py:clean_tarinfo
     @staticmethod
-    def _set_uid_gid(tarinfo: TarInfo) -> TarInfo:
+    def _set_uid_gid(tarinfo: TarInfo) -> TarInfo or None:
```

Comment:
This function name is increasingly inaccurate, it suggests it changes the uid and gid, but it also changed the mode, and now it filters undesirable files too!

I think it is long past time to change it, so maybe now would be a good opportunity to rename it to e.g. sanitize_tar.

Reply:
It sounds good. The name will make the function role clear.